app/misc/repo.ts

```diff
@@ -13,6 +13,7 @@ let span;
 let contributors: [any] = [0];
 let previousOpen;
 let repoName : string = "";
+let lastRefList = [];
```

Comment:
This variable serves the purpose of comparing newest fetched list to the last fetched one in refreshList().